docs/src/examples/gcode_fr.adoc

```diff
@@ -95,8 +91,6 @@ table d'outils.
 
 - Description: Dressage, filetage et tronçonnage.
 
-Ce programme donne un exemple de filetage automatique sur un tour avec
-utilisation des paramètres. Il demande quelques adaptations pour fonctionner, ces
-adaptations seront un excellent exercice.
-
-
+This program gives an example of automatic threading on a lathe with
+use of parameters. It requires some adaptations to work, these
+adaptations will be a great exercise.
```

Comment:
This translation is accurate, why delete ?

Reply:
I have no idea.